pkg/build/rpm.go

```diff
@@ -27,9 +36,23 @@ func (b *Builder) getRPMFileNames(rpmSourceDir string) ([]string, error) {
 	return rpmFileNames, nil
 }
 
-func (b *Builder) copyRPMs() error {
+func (b *Builder) copyRPMs(rpmSourceDir string, rpmDestDir string, rpmFileNames []string) error {
+	for _, rpm := range rpmFileNames {
+		sourcePath := filepath.Join(rpmSourceDir, rpm)
+		destPath := filepath.Join(rpmDestDir, rpm)
+
+		err := copyFile(sourcePath, destPath)
+		if err != nil {
+			return fmt.Errorf("copying file %s: %w", sourcePath, err)
+		}
+	}
+
+	return nil
+}
+
+func (b *Builder) processRPMs() error {
```

Comment:
Personal style note: I like to keep the main entry point (i'd normally say "public" if we weren't working with methods) at the top of the file, with the supporting methods (like copyRPMs) later on. But don't feel like you need to change it, just offering another perspective.

Reply:
I like that as well, I'll reorganize it